### PR TITLE
NFC: Fix typos in PackageLoading

### PR DESCRIPTION
**Pull request by MaxDesiatov**
The sentence in the second paragraph of `README.md` is unfinished. I'm not sure how it was supposed to look originally, so just cleaning it up as is.

### CODE REVIEW

**Comment by MaxDesiatov** [comment]
@swift-ci please smoke test

**Comment by MaxDesiatov** [comment]
@swift-ci please smoke test

File: Sources/PackageLoading/README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ This library defines the logic which translates between the Swift package
 manager conventions and the underlying project model.
 
 The intent is that it is largely a transformation taking the input project model
-objects descripted in a manifest, applying the conventions and
+objects descripted in a manifest, applying the conventions.
```
Comment (federicobucchi) [inline Sources/PackageLoading/README.md:7]: it: do we need the `,` comma?

Comment (MaxDesiatov) [reply Sources/PackageLoading/README.md:7]: Sorry, I'm not sure I follow, where exactly should the comma be added?

Comment (federicobucchi) [reply Sources/PackageLoading/README.md:7]: I mean, can we remove the comma?

`objects descripted in a manifest applying the conventions.`

Comment (MaxDesiatov) [reply Sources/PackageLoading/README.md]: That's fair, I somehow assumed that the rest should be kept verbatim 😄

Comment (federicobucchi) [reply Sources/PackageLoading/README.md]: thank you!

**Review by federicobucchi — APPROVED** [review]

**Comment by MaxDesiatov** [comment]
@swift-ci please smoke test

**Review by abertelrud — APPROVED** [review]

**Pull request merged**